qemu/tests/enforce_quit.py

```diff
@@ -21,6 +21,7 @@ def run(test, params, env):
     host_cpumodel = utils_misc.get_host_cpu_models()
     host_flags = utils_misc.get_cpu_flags()
     extra_flags = params.get("cpu_model_flags", " ")
+    no_check_flags = params.get("no_check_flags", " ").split()
```

Comment:
Hi @suqinhuang 
About "svm" flag ,how about this method ?
if "svm" in flags:
        force_quit = True